rules/best-practices.js

```diff
@@ -1,109 +1,199 @@
 
+    // Almost always results in nasty, hard to find bugs. Changing native/shared
+    // code can mean changes in expected behavior in other places.
     // http://eslint.org/docs/rules/no-extend-native
     'no-extend-native': 2,
 
+    // Sometimes during the course of code maintenance, the this value is
+    // removed from the function body. In that case, you can end up with a call
+    //  to bind() that doesn't accomplish anything. So save the performance.
     // http://eslint.org/docs/rules/no-extra-bind
     'no-extra-bind': 2,
 
+    // The switch statement in JavaScript is one of the more error-prone
+    // constructs of the language thanks in part to the ability to "fall
+    // through" from one case to the next. More often than not this is NOT
+    // what you want, so explicitly disable it.
     // http://eslint.org/docs/rules/no-fallthrough
     'no-fallthrough': 2,
 
-    // Don't allow people to do dumb things like "1." when they mean "1.0".
+    // Don't allow people to do silly things like "1." when they mean "1.0".
     // http://eslint.org/docs/rules/no-floating-decimal
     'no-floating-decimal': 2,
 
+    // There are some other ways to pass a string and have it interpreted as
+    // code. Disable them for the same reasons as `eval`.
     // http://eslint.org/docs/rules/no-implied-eval
     'no-implied-eval': 2,
 
+    // In JavaScript, prior to ES6, standalone code blocks delimited by curly
+    // braces do not create a new scope and have no use.
     // http://eslint.org/docs/rules/no-lone-blocks
     'no-lone-blocks': 2,
 
+    // Writing functions within loops tends to result in errors due to the way
+    // the function creates a closure around the loop. If functions are needed
+    // in loops, use `let` as the iterator variable or use an iterator function
+    // like `forEach`.
     // http://eslint.org/docs/rules/no-loop-func
     'no-loop-func': 2,
 
+    // It's possible to create multiline strings in JavaScript by using a slash
+    // before a newline. Use template strings instead.
     // http://eslint.org/docs/rules/no-multi-str
     'no-multi-str': 2,
 
+    // Reports an error when they encounter an attempt to assign a value to
+    // built-in native object. There should be no valid reason anyone is trying
+    // to assign values to built-ins.
     // http://eslint.org/docs/rules/no-native-reassign
     'no-native-reassign': 2,
 
+    // Don't use `new` for side-effects. Since `new` always creates an object
+    // instances, not consuming that object is wasteful and generally indicative
+    // of programmer error. Sometimes this is handy in testing and in those
+    // cases exceptions can be made.
     // http://eslint.org/docs/rules/no-new
     'no-new': 2,
 
+    // It's possible to create functions in JavaScript using the Function
+    // constructor. This is considered by many to be a bad practice due to the
+    // difficult in debugging and reading these types of functions.
```

Comment:
Little grammar mistake (`due to the difficult`). Perhaps something like:
`This is considered by many to be a bad practice as it increases debugging difficultly`